libs/application/templates/children-residence-change/src/fields/ContactInfo/index.tsx

```diff
@@ -30,22 +35,24 @@ const ContactInfo = ({ error, application, field }: FieldBaseProps) => {
       </Box>
       <Box marginTop={5}>
         <Controller
-          name="email"
-          defaultValue={getValue('contactInfo')?.[0]}
+          name={emailId}
+          defaultValue={getValue(phoneNumberId)}
```

Comment:
Should this be `emailId` ?

Reply:
I though about it but decided to go with the same approach as all the wrapper controller comopnents - same name for the controller and the input fields:
https://github.com/island-is/island.is/blob/main/libs/shared/form-fields/src/lib/InputController/InputController.tsx#L143